### ISSUE TITLE
Question in total GAN loss

### ISSUE THREAD

**Issue by Yoan-D** [clovaai/stargan-v2#46]
Hi,

thanks for your paper and code. I was wondering if you explain why you defined your total GAN loss as: loss = loss_adv + args.lambda_sty * loss_sty - args.lambda_ds * loss_ds + args.lambda_cyc * loss_cyc. Why did you combine the GAN loss and style loss and subtract it from the diversity and cyclic loss? What is the intuition behind that?

Thanks for your time!

**Comment by Johnson-yue** [comment]
the reason in paper

**Comment by Yoan-D** [comment]
Aha.
